style.css

```diff
@@ -6,6 +6,30 @@
   box-sizing: border-box;
 }
 
+body {
+  max-width: 1440px;
+  margin: 0 auto;
+}
+
+ul li {
+  list-style: none;
+}
+
+a {
+  /*
+  width: 59px;
+  height: 20px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 20px;
+  */
```

Comment:
Please do not leave commented pieces of code in your code base and they clutter it and make it less readable.